src/plugins/unified_histogram/public/container/hooks/use_state_props.ts

```diff
@@ -86,14 +91,29 @@ export const useStateProps = ({
   }, [chartHidden, isPlainRecord, isTimeBased, timeInterval]);
 
   const breakdown = useMemo(() => {
-    if (isPlainRecord || !isTimeBased) {
+    if (!isTimeBased) {
       return undefined;
     }
 
+    // hide the breakdown field selector when the ES|QL query has a transformational command (STATS, KEEP etc)
+    if (query && isOfAggregateQueryType(query) && hasTransformationalCommand(query.esql)) {
+      return undefined;
+    }
+
+    if (isPlainRecord) {
+      const breakdownColumn = columns?.find((column) => column.name === breakdownField);
+      const field = breakdownColumn
+        ? new DataViewField(convertDatatableColumnToDataViewFieldSpec(breakdownColumn))
+        : undefined;
+      return {
+        field,
+      };
```

Comment:
This is actually different then returning `undefined` as before. Now we are returning `{field: undefined`.
Just want to check if this is the desired behaviour, because if I search for that `breakdown` is used for example in a `charts.tsx`:
```
{chartVisible && breakdown && (
                    <BreakdownFieldSelector
                      dataView={dataView}
                      breakdown={breakdown}
                      onBreakdownFieldChange={onBreakdownFieldChange}
                    />
                  )}
```
that hypothetical create the `BreakdownFieldSelector` component,  that was different then the previous behaviour

Reply:
yes it is on purpose! I am following the exact same logic as dataview mode (check the line below)

Reply:
sounds good, thanks